### DISCUSSION TITLE
5 minute synchronization task failing with dropbox enabled for storage

### DISCUSSION THREAD

**Discussion by doctator81** [AzuraCast/AzuraCast#4106] [Q&A]
Hi all, 

So we're trying to host most if not all our media out of dropbox. Trouble is, whenever we have dropbox enabled for storage, the 5 minute sync task seems to time out. When I increase the execution time in azuracast.env it doesn't seem to run at all. Any ideas?

**Comment by MadHitzFM** [comment]
Hi! Try reverting the changes you made in azuracast.env file, running the task in the debug area.
It might be worth updating your installation, even if you're on the latest one, just incase something didn't download or pull correctly.

If you're still having issues, post the error here, and it could help debugging
